Jenkinsfile-insertions

```diff
@@ -69,12 +69,10 @@ pipeline{
 		stage('Post: Backup release_current'){
 			steps{
 				script{
-					dir('add-links'){
-						withCredentials([usernamePassword(credentialsId: 'mySQLUsernamePassword', passwordVariable: 'pass', usernameVariable: 'user')]){
-							def release_current_after_addlinks_dump = "${env.RELEASE_CURRENT}_${currentRelease}_after_addlinks.dump"
-							sh "mysqldump -u$user -p$pass ${env.RELEASE_CURRENT} > $release_current_after_addlinks_dump"
-							sh "gzip $release_current_after_addlinks_dump"
-						}
+					withCredentials([usernamePassword(credentialsId: 'mySQLUsernamePassword', passwordVariable: 'pass', usernameVariable: 'user')]){
+						def release_current_after_addlinks_dump = "${env.RELEASE_CURRENT}_${currentRelease}_after_addlinks.dump"
+						sh "mysqldump -u$user -p$pass ${env.RELEASE_CURRENT} > $release_current_after_addlinks_dump"
```

Comment:
I think the new common Groovy library has a function to do mysqldump and gzip that could be used here.

Reply:
This needs to get merged before the updated code to use the new shared library gets merged. Otherwise this branch would _never_ get merged.

Reply:
This is an old PR so consider the JF work to be irrelevant as I'll be updating it immediately following the merge of this branch